src/App.tsx

```diff
@@ -5,7 +5,7 @@ import { LoginPage, RegisterPage } from './components';
 export default function App() {
   const dummyData = {
     title: 'Naver',
-    favicon: '',
+    favicon: 'https://s.pstatic.net/static/www/mobile/edit/2016/0705/mobile_212852414260.png',
```

Comment:
Favicon URL of [naver](www.naver.com)
<img width="909" alt="스크린샷 2020-08-29 오후 10 37 10" src="https://user-images.githubusercontent.com/14817195/91638131-2f47c900-ea48-11ea-8b94-f4c5286a1b60.png">

Reply:
It seems that dummy data should be written as test code later.